src/types/log.rs

```diff
@@ -93,6 +74,25 @@ pub struct Filter {
     limit: Option<usize>,
 }
 
+#[derive(Default, Debug, PartialEq, Clone)]
```

Comment:
Why was it moved? I think I preferred when `Filter` and `FilterBuilder` were next to each other.